app/src/main/res/layout/pad_operator_one_col.xml

```diff
@@ -23,20 +23,19 @@
     android:columnCount="1"
     android:rowCount="5">
 
-    <com.android.calculator2.view.AutoFitButtonView
+    <ImageButton
         android:id="@+id/del"
-        style="@style/PadButtonStyle.Operator.Text"
+        style="@style/PadButtonStyle.Icon"
         android:contentDescription="@string/desc_del"
         android:onClick="onButtonClick"
-        android:singleLine="true"
-        android:text="@string/del" />
+        android:src="@android:drawable/ic_menu_delete" />
```

Comment:
@gscdev Since all of our tablets are running on Android API 21 or higher, I recommend we use vectors instead of image files. This reduces file size and keeps the resolution while scaling to different screen sizes.

```java
android:src="@drawable/ic_delete_black_24dp"
android:tint="#FFF"
```

See https://www.youtube.com/watch?v=8e3I-PYJNHg

![screenshot_20161103-093017](https://cloud.githubusercontent.com/assets/1451036/19957935/fb976432-a1ab-11e6-97a0-8b94d56c5b34.png)
![screenshot_20161103-093029](https://cloud.githubusercontent.com/assets/1451036/19957938/fbf4b3ee-a1ab-11e6-886a-c877890e5b86.png)
![screenshot_20161103-094953](https://cloud.githubusercontent.com/assets/1451036/19957939/fbf6cd8c-a1ab-11e6-8052-aae9146c285e.png)
![screenshot_20161103-094959](https://cloud.githubusercontent.com/assets/1451036/19957937/fbf3db72-a1ab-11e6-9ef0-2489b5141528.png)